src/mscorlib/src/System/Runtime/InteropServices/IException.cs

```diff
@@ -23,7 +23,7 @@ namespace System.Runtime.InteropServices {
     [InterfaceTypeAttribute(ComInterfaceType.InterfaceIsDual)]
     [CLSCompliant(false)]
     [System.Runtime.InteropServices.ComVisible(true)]
-    public interface _Exception
```

Comment:
This whole file should be deleted - I forgot to delete it when I was cleaning up these.